src/EnergyPlus/PlantLoopHeatPumpEIR.cc

```diff
@@ -3058,4 +3171,19 @@ void EIRFuelFiredHeatPump::oneTimeInit(EnergyPlusData &state)
     }
 }
 
+void EIRFuelFiredHeatPump::report(EnergyPlusData &state)
+{
+    Real64 const reportingInterval = state.dataHVACGlobal->TimeStepSysSec;
+
+    this->fuelEnergy = this->fuelRate * reportingInterval;
+    this->loadSideEnergy = this->loadSideHeatTransfer * reportingInterval;
+    this->powerEnergy = this->powerUsage * reportingInterval;
+    this->sourceSideEnergy = this->sourceSideHeatTransfer * reportingInterval;
+
+    // update nodes
+    PlantUtilities::SafeCopyPlantNode(state, this->loadSideNodes.inlet, this->loadSideNodes.outlet);
+    state.dataLoopNodes->Node(this->loadSideNodes.outlet).Temp = this->loadSideOutletTemp;
+    state.dataLoopNodes->Node(this->sourceSideNodes.outlet).Temp = this->sourceSideOutletTemp;
+}
+
```

Comment:
Add similar new member report() function for `EIRFuelFiredHeatPump` module as well.